frontend/test/pytest/test_preprocess.py

```diff
@@ -171,6 +225,31 @@ def circuit(theta: float):
         assert "var" not in mlir
         assert "counts" in mlir
 
+    @pytest.mark.skipif(
+        not pathlib.Path(
+            get_lib_path("runtime", "RUNTIME_LIB_DIR") + "/libdummy_device.so"
+        ).is_file(),
+        reason="lib_dummydevice.so was not found.",
+    )
```

Comment:
Would be great to ensure this always runs, not only when the optional dummy component is available.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/702

Reply:
Merged